storybook/storyLoader.js

```diff
@@ -0,0 +1,21 @@
+// Auto-generated file created by react-native-storybook-loader
+// Do not edit.
+//
+// https://github.com/elderfo/react-native-storybook-loader.git
+
+function loadStories() {
+  require('../src/components/atoms/button/stories/button.stories.common');
+  require('../src/components/atoms/button/stories/button.stories.native');
+  require('../src/components/organisms/navigation/stories/nav.stories.native');
+}
+
+const stories = [
+  '../src/components/atoms/button/stories/button.stories.common',
+  '../src/components/atoms/button/stories/button.stories.native',
+  '../src/components/organisms/navigation/stories/nav.stories.native'
+];
+
+module.exports = {
+  loadStories,
+  stories
+};
```

Comment:
Is there any need to check this in as it will be auto-generated each time we run the Storybook?

Reply:
@jh3y no need to check this file in, I'll remove it ! Good catch @jh3y